apps/web/composables/useItemTable/useItemTable.ts

```diff
@@ -0,0 +1,79 @@
+import type { UseItemTableState, UseItemTableReturn } from './types';
+import type { StorageObject } from '@plentymarkets/shop-api';
+
+export const useItemsTable: UseItemTableReturn = () => {
+  const state = useState<UseItemTableState>('useItemTable', () => ({
+    data: [],
+    loading: false,
+  }));
+
+  const cachedImages = useState<StorageObject[]>('image-table-cache', () => []);
+
+  const getStorageItemsServer = async (fileTypes = 'png,jpg,jpeg,avif,webp') => {
+    state.value.loading = true;
+
+    if (cachedImages.value.length > 0) {
+      state.value.data = cachedImages.value;
+      state.value.loading = false;
+      return;
+    }
+
+    const { data, error } = await useAsyncData(() =>
+      useSdk().plentysystems.getStorageItems({
+        fileTypes,
+      }),
+    );
+    state.value.loading = false;
+
+    if (error.value) {
+      const { send } = useNotification();
+      send({ type: 'negative', message: error.value.message });
+      return;
+    }
+
+    const items: StorageObject[] = data?.value?.data ?? [];
+
+    state.value.data = items;
+    cachedImages.value = items;
+  };
+
+  const headers = ref([
+    { title: 'File name', key: 'key' },
+    { title: 'Image size', key: 'size' },
+    { title: 'Last change', key: 'lastModified' },
+  ]);
+
+  const bytesToMB = (bytes: string | number): string => {
+    const num = typeof bytes === 'string' ? parseInt(bytes, 10) : bytes;
+    return (num / (1024 * 1024)).toFixed(2) + ' MB';
+  };
+
+  const formatDate = (dateStr: string): string => {
+    if (!dateStr) return '';
+    const date = new Date(dateStr);
+    return date.toLocaleString('en-US', {
```

Comment:
```suggestion
    return date.toLocaleString('en-GB', {
```
I believe we still use British English officially.